docs/public/SUMMARY.md

```diff
@@ -26,8 +27,13 @@
     - [Profiles](configuration/profiles.md)
     - [Global Cache](configuration/global-cache.md)
     - [Backend](configuration/backend.md)
-        - [Environment Variables](configuration/backend-environment-variables.md)
-    - [GUI](configuration/gui.md)
+    - [Environment Variables](configuration/backend-environment-variables.md)
```

Comment:
This change should be reverted. The document relates solely to Backend, so the indentation is needed.